bindings/matrix-sdk-crypto-js/src/types.rs

```diff
@@ -84,6 +91,42 @@ impl Signatures {
     pub fn count(&self) -> usize {
         self.inner.signature_count()
     }
+
+    /// Get the json with all signatures
+    #[wasm_bindgen(js_name = "asJSON")]
+    pub fn as_json(&self) -> JsValue {
+        trace!(?self.inner, "The signature");
+        // can't use directly serde_wasm_bindgen as there is an issue with BTreeMap
+        // It's always returning an empty {} if I do:
```

Comment:
Hmm, but you collect into a `BTreeMap` a couple of lines bellow as well, in fact the collect is almost the same thing the `Serialize` implementation does for the `Signatures` struct.

If `serde_wasm_bindgen` didn't work, did `serde_json::to_string()` work correctly? I don't see how that would be different to what you're doing here.